lib/ical_filter_proxy/calendar_builder.rb

```diff
@@ -33,5 +34,21 @@ def add_rules
       end
     end
 
+    def add_alarms
+      alarms = calendar_config["alarms"]
+      return unless alarms
+
+      if alarms["clear_existing"] == true
```

Comment:
A more idiomatic statement here would be

```ruby
calendar.clear_existing_alarms = true if alarms['clear_existing']
```

Or you could even go to:

```ruby
calendar.clear_existing_alarms = !!alarms['clear_existing']
```

Reply:
As a non-ruby native developer, first option is more understandable for me.
Done.